### ISSUE TITLE
The windows' close button has duplicate tooltips in both English and Local language

### ISSUE THREAD

**Issue by pohuing** [microsoft/terminal#14951]
### Windows Terminal version

 1.16.10262.0

### Windows build number

10.0.22621.0

### Steps to reproduce

Hover the center of the windows' close button.

### Expected Behavior

A tooltip should appear in the local language's equivalent of "Close".

### Actual Behavior

Hovering the button shows two tooltips fade in, first the English one, then the local language's one.

![image](https://user-images.githubusercontent.com/24683548/222844614-a1996e5e-7ff6-41bb-ae21-836e01730ecc.png)

https://user-images.githubusercontent.com/24683548/222845299-ac3d37c1-e5fb-4a34-997a-f71d086ca9cd.mp4

**Comment by pohuing** [comment]
Sidenote: This only happens on the close button. None of the other buttons are translated to German. Maybe because localization in general only works after changing the setting back and forth 🤔

**Comment by zadjii-msft** [comment]
Okay so generally, this is a dupe of #12882. It's a really wacky platform bug, for sure. Interesting to me that the second one is all Mica and WinUI-y and rejuvenated - for me I still get the "yellow" win32 style one. 

There is one interesting note here though:

> first the English one, then the local language's one

That's weird to me, because we're displaying the first tooltip. Is the rest of the Terminal in German/? (Settings tab in particular has a lot of strings). _Maybe_ those few strings didn't get localized, but that doesn't really make sense to me. I'm pretty confident the language doesn't hot-reload, so if you did change it in the Terminal settings, you'd need to quit & re-open.

**Comment by zadjii-msft** [comment]
what the sweet baby jesus

`MinMaxCloseControl.xaml`:
![image](https://user-images.githubusercontent.com/18356694/223423179-6accd93c-88b8-4365-8970-7f8c97599e8e.png)
`TerminalApp/Resources.resw`:
![image](https://user-images.githubusercontent.com/18356694/223423300-738a6f6e-9596-4716-bfbb-62c4bfec6fea.png)

but also, all three buttons have both a `ToolTipService.ToolTip` in the `.resw` AND the XAML 


except NO THEY DON'T

`resw`:
![image](https://user-images.githubusercontent.com/18356694/223423818-d962dc1f-425f-44cb-bc4e-1afedbf82a72.png)

`xaml`:
![image](https://user-images.githubusercontent.com/18356694/223423984-c87b7be3-0f20-4c31-9f5b-fc9c28fae334.png)

`WindowCloseButton` != `CloseButton`

what the heck is going on here

**Comment by pohuing** [comment]
https://user-images.githubusercontent.com/24683548/223426187-c952e700-5f86-4097-b424-f32d5637768c.mp4

The localisation option just does not work. Note in the last 15 seconds how it switches back and forth between Germand and English, but only sometimes? And applying the same setting twice yields different results.

I should note that I am using StartAllBack, which messes with some system UI parts, I'll test if those might cause the issue.

**Comment by pohuing** [comment]
Unfortunately removing StartAllBack doesn't change anything. The localisation option is still broken

**Comment by zadjii-msft** [comment]
Okay so couple thoughts:

* Changing the language in the Terminal settings does typically require a full restart of the Terminal to properly take effect. 
* Do you have something like MicaForEveryone installed? We're a little confounded as to how the second tooltip appeared with the Mica effect, cause that one we don't control. But that's more out of curiosity.
* The "double popups" is tracked in #12882
* We're gonna use this to track cleaning up the resw which is clearly insane

**Comment by pohuing** [comment]
> Do you have something like MicaForEveryone installed?

No, but I wouldn't rule out if StartAllBack or an alternative's leftovers were messing with that. I tried a couple things to get some functionality back when I switched to w11. Unfortunately I don't have a free drive to try a clean windows installation.

**Comment by pohuing** [comment]
Okay sorry. I told windows to give applications the en_GB locale. Meaning my complaints about the incorrect system locale were nonsense. I've gotten so used to applications not listening to that setting, that I never noticed that they all showed the wrong locale.
And I should have noticed that in the example video as well: this says to prefer English over German in Windows Store Applications.
![image](https://user-images.githubusercontent.com/24683548/224491828-20677c3a-f9dd-46bf-9744-b3f13c8a4eca.png)

Meaning aside from the shells not using the application's locale and the slightly flaky switching near the end in the video above, the localization is working as intended.

The double tooltip still shows though: 
![image](https://user-images.githubusercontent.com/24683548/224491920-c7f2baf0-f8df-484d-8b70-531ee7af47bd.png)
